odoo/models.py

```diff
@@ -5771,7 +5771,7 @@ def invalidate_cache(self, fnames=None, ids=None):
                [(invf, None) for f in fields for invf in self._field_inverses[f]]
         self.env.cache.invalidate(spec)
 
-    def modified(self, fnames, create=False):
+    def modified(self, fnames, create=False, before=False):
```

Comment:
Could you update the docstring to also document the `before` parameter ? :)
`:param bool before: ...`